modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -147,8 +182,9 @@ public function verify_frame_nonce( $nonce, $action ) {
 	/**
 	 * Filters the WordPress salt.
 	 *
-	 * @param string $salt   Salt for the given scheme.
+	 * @param string $salt Salt for the given scheme.
 	 * @param string $scheme Authentication scheme.
+	 *
```

Comment:
@mmtr Are these whitespace changes prompted by phpcs?

Reply:
Oops, I didn't notice this. Yes, I performed an automatic reformat code on the file while I wanted to run in only in a specific piece of code and it seems that this space was added automatically because my IDE thinks that it should be a new line before the `@return` description. 

I don't see that's required on https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/ so I'm happy to revert that change back.

Reply:
Actually, it seems like we use this pattern (separate `@return` from `@param`) in many Jetpack functions, so probably it is ok to leave it as it is?

Reply:
I'd bet the separation stems from IDE's default settings. Core groups `@param` and `@return` notations and aligns `@param` descriptions.
It's not terribly important in the scheme of things, and I'm not sure if Jetpack has diverging coding standards from Core's that would support it, but we get around to it I think it would be worth not changing.